pkg/kotsadm/main.go

```diff
@@ -165,21 +166,21 @@ func Deploy(deployOptions types.DeployOptions, log *logger.CLILogger) error {
 		}
 
 		if !deployOptions.DisableImagePush {
-			err := image.TagAndPushImagesFromPath(deployOptions.AirgapRootDir, pushOptions)
+			err := image.TagAndPushImagesFromBundle(deployOptions.AirgapBundle, pushOptions)
 			if err != nil {
 				return errors.Wrap(err, "failed to tag and push app images from path")
 			}
 		}
 
-		airgapPath = deployOptions.AirgapRootDir
+		airgapPath = deployOptions.AirgapBundle
```

Comment:
how about reusing `deployOptions.AppImagesPushed = true` here and remove this `airgapPath` variable that is only used to indicate the app images were pushed? And use `deployOptions.AirgapBundle` when getting the `airgap.yaml` file.

Reply:
hmm, i do see `deployOptions.DisableImagePush` so it might be confusing... nevermind

Reply:
addressed this in https://github.com/replicatedhq/kots/pull/4518/commits/07e0d7bf117ec9c5ae64de657a7a67e312297423

Reply:
and https://github.com/replicatedhq/kots/pull/4518/commits/2be05f26e568f7c7747b6552bc9431e8344ef762 :)